modules/oadp-self-signed-certificate.adoc

```diff
@@ -39,3 +39,29 @@ spec:
 ----
 <1> Specify the Base64-encoded CA certificate string.
 <2> The `insecureSkipTLSVerify` configuration can be set to either `"true"` or `"false"`. If set to `"true"`, SSL/TLS security is disabled. If set to `"false"`, SSL/TLS security is enabled.
+
+[id="oadp-using-ca-certificates-with-velero-command-aliased-for-velero-deployment_{context}"]
+== Using CA certificates with velero command aliased for velero deployment
+
+You might want to use the Velero CLI without installing it locally on your system.
+
+If you want to use the Velero CLI without installing it locally you can do so by using an aliased Velero command, such as the following command:
+
+[source,terminal]
+----
+alias velero='oc -n openshift-adp exec deployment/velero -c velero -it -- ./velero'
+----
+
+If you want to use CA certificate with this command, you can add a certificate to the Velero deployment by running the following commands:
+
+[source,terminal]
+----
+$ oc get dataprotectionapplications.oadp.openshift.io <dpa-name> -o jsonpath='{.spec.backupLocations[0].velero.objectStorage.caCert}' | base64 -d | oc exec -i deploy/velero -c velero -- bash -c "cat > /tmp/your-cacert.txt"
```

Comment:
I think we are missing here the namespace from which dpa is used:

```shell
$ oc -n openshift-adp get dataprotectionapplications.oadp.openshift.io <dpa-name> -o jsonpath='{.spec.backupLocations[0].velero.objectStorage.caCert}' | base64 -d | oc exec -i deploy/velero -c velero -- bash -c "cat > /tmp/your-cacert.txt"
```

Secondly if the DPA does not have any `.spec.backupLocations[0].velero.objectStorage.caCert` then the entire command will fail, maybe it's worth adding it by printing error when it's not there.
For readability I would split this into few shell lines:


```bash
$ CA_CERT=$(oc -n openshift-adp get dataprotectionapplications.oadp.openshift.io <dpa-name> -o jsonpath='{.spec.backupLocations[0].velero.objectStorage.caCert}'

$ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i deploy/velero -c velero -- bash -c "cat > /tmp/your-cacert.txt" || echo "DPAs' BSL has no caCert"
```

Reply:
Many thanks. I would like other person to approve my suggestions and text, especially in the `echo "DPAs' BSL has no caCert"` part.